lib/main_containers.pm

```diff
@@ -67,9 +67,9 @@ sub load_host_tests_podman {
 
 sub load_host_tests_docker {
     loadtest 'containers/docker';
-    loadtest 'containers/docker_image';
+    loadtest 'containers/docker_image' unless (is_public_cloud());
     loadtest 'containers/docker_3rd_party_images';
-    unless (is_sle("<=15") && is_aarch64) {
+    if (is_opensuse() || (is_sle(">15") && !is_aarch64())) {
```

Comment:
Is this the same as the condition you used above `is_sle("15-sp1+")` ? If so, we could use the same expression for clarity.